.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, review_requested]
+  issue_comment:
+    types: [created]
+
+jobs:
+  code_review_job:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    name: Run code review agent on every pull request, respond to user comments
+
+    steps:
+      - name: Set Options Environment Variable for Issue Comment
+        if: github.event_name == 'issue_comment'
+        run: |
+          echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=/automation-platform  --bee.actn_dir=/automation-platform/default_bito_ad/bito_modules --git.access_token=${{ secrets.GIT_ACCESS_TOKEN }} --bito_cli.bito.access_key=${{ secrets.BITO_ACCESS_KEY }} ${{ secrets.GIT_DOMAIN && ' --git.domain=' || '' }}${{ secrets.GIT_DOMAIN }}" >> $GITHUB_ENV
```

Comment:
**Security Issue**: The workflow script includes potentially sensitive tokens as environment variables without clear encryption or masking, which could lead to accidental exposure in logs or to unauthorized users. <br> **Fix**: Use GitHub secrets to store sensitive information and access them securely within the workflow. Ensure that any sensitive environment variables are accessed securely and are not explicitly echoed or logged. <br> **Code Suggestion**: 
 ```
 jobs:
   code_review_job:
     steps:
       - name: Set Options Environment Variable for Pull Request
         run: |
           echo "GIT_ACCESS_TOKEN=${{ secrets.GIT_ACCESS_TOKEN }}" >> $GITHUB_ENV
           echo "BITO_ACCESS_KEY=${{ secrets.BITO_ACCESS_KEY }}" >> $GITHUB_ENV
           # Ensure secrets are not logged
         env:
           GIT_ACCESS_TOKEN: ${{ secrets.GIT_ACCESS_TOKEN }}
           BITO_ACCESS_KEY: ${{ secrets.BITO_ACCESS_KEY }}
 ```